### ISSUE TITLE
Columns reset index if data in them overflows current page.

### ISSUE THREAD

**Issue by krodyrobi** [ralfstuckert/pdfbox-layout#59]
Consider having 2 columns (labels and data) if data overflows the current page it won't continue on column 2 on the next page instead it will start on column 1.

Is there a way to prevent this?

**Comment by ralfstuckert** [comment]
This would be the table feature which is requested by many people. Alas, currently I don't have much time to work on this.
Sorry

**Comment by arnthom** [comment]
I have a similar problem and tried this workaround:
- switch to column layout
- define columns
- write text elements of row 1 into columns 
- switch to vertical layout
- repeat with row 2 and so on

This will ensure, that end of column 2 does not go into column 1.
Do you need some code for this workaround?

**Comment by krodyrobi** [comment]
If it wouldn't cause too much trouble I'd greatly appreciate it :D.

**Comment by krodyrobi** [comment]
Thank you!
Will have a look and see.

**Comment by krodyrobi** [comment]
Yes it works and you've set me in a cleaner direction, thank you again!
